### DISCUSSION TITLE
SkyeTracker Android App compatibity

### DISCUSSION THREAD

**Discussion by Nickous** [ClassicDIY/SkyeTracker#43] [General]
Hello Graham
I discovered your project last month and I'm very excited about the idea of making my SkyeTracker.
I see the Android app...but it's not compatible with my phone (I guess it's because I'm on A13 version ?!).
I was wondering if you were planning to make any compatibity changes within the next 6 months?
(I have no idea how to program in Android Studio, but I can learn if necessary :))
Thanks

**Comment by graham22** [comment]
I published an update to google play but it's stuck in some review process.
if you want, you can sideload the apk from my dropbox at
https://www.dropbox.com/scl/fi/z9q8ongcaypl73jyujbsl/app-release.apk?rlkey=eayt09bo2x7m306dtuwc95mtg&dl=0
You will need to allow unknown apps in your settings...



Graham

On Fri, Sep 8, 2023 at 10:58 AM Nickous ***@***.***> wrote:

> Hello Graham
> I discovered your project last month and I'm very excited about the idea
> of making my SkyeTracker.
> I see the Android app...but it's not compatible with my phone (I guess
> it's because I'm on A13 version ?!).
> I was wondering if you were planning to make any compatibity changes
> within the next 6 months?
> (I have no idea how to program in Android Studio, but I can learn if
> necessary :))
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/43>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATISDRPURDNGULHMX5LXZMXALANCNFSM6AAAAAA4QQKEOQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331

**Comment by Nickous** [comment]
Amazing!
I'm really not ready to use it and test my SkyeTracker but it's quietly maturing in my mind.
Thank you very much

**Comment by graham22** [comment]
There are some issues with the app using the new google API, will publish a fix in a couple of days

**Reply by Nickous** [reply]
Hello
Just to confirm you that now new update 2.5.6 is available on Google Store for my phone 
Thank you
